frontend/src/deadlineList/components/userAvatar.jsx

```diff
@@ -13,10 +13,20 @@ const UserAvatar = ({ width, src, ...props }) => {
 			case 3:
 				translateX = 'transform -translate-x-3/2'
 		}
+		let border = 0
```

Comment:
hey instead of only changing the numeric after the border change the whole border write up e.g border="border-2"